service/alerter/service.go

```diff
@@ -204,6 +233,33 @@ func FindOrphanResources(clusterIDs []string, resourceNames []string) []string {
 	return orphanResources
 }
 
+// FindOrphanClusters compares a list of cluster IDs and resource names. It
+// returns the IDs of any cluster created a reasonable amount of time ago without
+// resources associated.
+func FindOrphanClusters(awsConfigs []v1alpha1.AWSConfig, resourceNames []string) []string {
+	resources := map[string]bool{}
+	orphanClusters := []string{}
+
+	for _, resourceName := range resourceNames {
+		resources[resourceName] = true
+	}
+
+	n := time.Now()
+	l := n.Add(-10 * time.Minute)
```

Comment:
I like the delay here to prevent false positives / alert spam. What about extracting the time interval to a constant? If we add more alerts it would be good to use the same delay.

Reply:
Good point, done.